packages/lib-classifier/src/components/Classifier/components/Banners/components/Banner/Banner.js

```diff
@@ -64,54 +52,51 @@ class Banner extends Component {
       tooltipText
     } = this.props
 
-    return (
-      <PoseGroup>
-        {show && (
-          <PosedBox
-            align='center'
-            background={{ color: background, opacity: 'strong' }}
-            className={className}
-            direction='row'
-            key={bannerText}
-            justify='between'
-            pad='small'
-            show={show}
+    return show && (
+      <Box
+        align='center'
+        animation='fadeIn'
+        background={{ color: background }}
+        className={className}
```

Comment:
`classname` is probably no longer necessary, since it's using the standard `Box` component now.